docs/openapi-fetch/examples.md

```diff
@@ -4,130 +4,28 @@ title: openapi-fetch Examples
 
 # Examples
 
-## Authentication
```

Comment:
With middleware, these awkward, one-off workarounds can go away. Now it can be handled more-automatically.